tests/conftest.py

```diff
@@ -52,12 +53,22 @@ def complete_installation(inventory):
 
 
 @when(parsers.parse("I launch ansible with the '{playbook}' playbook"))
-def ansible_playbook_step(request, inventory, playbook):
+def ansible_playbook_step(request, playbook):
     ansible_process = run_ansible_playbook(playbook)
     request.ansible_process = ansible_process
     return ansible_process
 
 
+@when(parsers.parse("I redeploy '{tag}'"))
+def redeploy_tag(request, tag):
+    ansible_process = run_ansible_playbook(
+        "deploy.yml",
+        external_vars={'helm_wait': True},
```

Comment:
Seems that `helm upgrade --install --wait` is quite buggy

Reply:
How so?

Reply:
fatal: [<redacted>]: FAILED! => {"changed": false, "msg": "Error running helm --install nginx-ingress upgrade nginx-ingress --version 0.23.0 --repo https://kubernetes-charts.storage.googleapis.com --
-ingress --wait --values /tmp/ansible.helm.values.Uhtyl6.yaml --values /tmp/ansible.helm.values.qUoqmC.yaml command (rc=1) out='' err='Error: UPGRADE FAILED: timed out waiting for the condition\n'"

frequency: 100%